pygmt/src/velo.py

```diff
@@ -0,0 +1,252 @@
+"""
+velo - Plot velocity vectors, crosses, anisotropy bars and wedges.
+"""
+import numpy as np
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    A="vector",
+    B="frame",
+    C="cmap",
+    D="rescale",
+    E="uncertainty_color",
+    G="facecolor",
+    H="scale",
+    I="intensity",
+    J="projection",
+    L="line",
+    N="no_clip",
+    R="region",
+    S="scaling",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    Z="zvalue",
+    c="panel",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
+def velo(self, data=None, **kwargs):
+    r"""
+    Plot velocity vectors, crosses, anisotropy bars and wedges.
+
+    Reads data values from files, numpy array or pandas DataFrame and will plot
```

Comment:
```suggestion
    Reads data values from files, numpy arrays or pandas DataFrames and will plot
```

Reply:
If it's not too much work, I think that formatting these using intersphinx mapping would be the most useful here and in line 56 (`data` parameter description).

Reply:
Cool, intersphinx mappings added in d67bfcdfe0ed90b97d721eaec17fccd348c3d07f.